drivers/pds/dataservice/workloads.go

```diff
@@ -0,0 +1,367 @@
+package dataservice
+
+import (
+	"fmt"
+	pds "github.com/portworx/pds-api-go-client/pds/v1alpha1"
+	pdsdriver "github.com/portworx/torpedo/drivers/pds"
+	"github.com/portworx/torpedo/pkg/log"
+	v1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"strings"
+	"time"
+)
+
+func CreateServiceAccount(namespace string) (*corev1.ServiceAccount, error) {
```

Comment:
Please make this function private , if its only utilized by generate function. Else kindly move it to k8s utilities.

Reply:
postgresql-datavalidation